src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Tooltip, { TooltipPosition } from '../tooltip';
+
+import './LabelPill.scss';
+
+export enum LabelPillStatus {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
+    tooltip?: string;
+    type?: LabelPillStatus;
+}
+
+const LabelPill = ({ children, type = LabelPillStatus.DEFAULT, tooltip = '', ...rest }: LabelPillProps) => {
+    const labelPillClasses = classNames(
+        'bdl-LabelPill',
+        type !== LabelPillStatus.DEFAULT ? `bdl-LabelPill--${type}` : '',
```

Comment:
```suggestion
{ [`bdl-LabelPill--${type}`]: type !== LabelPillStatus.DEFAULT },
```

FYI this is the "classnames" way. It will drop the `key: value` pairs where values are false.

Reply:
Should this accept className as a prop, too? And forwardRef so it can work with tooltips and stuff?

Reply:
It looks gross, but is more terse and avoids generating an empty string. ++

I avoided adding a classname since the pill size, shape, and colors should be stable (should force people to either try and override the class names of the component directly (BAD), or ask questions (good).)

I don't know about `forwardRef`. I don't foresee this being more than just a visual component, and any behaviors will be added thru composition. Then again, the ref stuff confuses me so I don't know

Reply:
it does have `...rest` so I suppose a person could add `className="blah"` if they really wanted to

Reply:
That would override the original classNames, removing the styles. Usually we pass through classname in order to add layout classes, mostly flex or margins. I agree we don't expect colors to be customized here through CSS overrides.

Reply:
I think we need forwardRef for all "leaf nodes" of the design system, so that you can get a ref to a DOM node to attach listeners and work with wrappers like Popper/the new version of Tether.

Reply:
We don't seem to be doing this now, as most of the existing components with any reference to fowardRef are some form of HOC. All of the other components with forwardRef are controls (select button, dropdown, nav button). I don't know the exact pattern with this, but this component doesn't seem to fit with the other things that implement forwarding refs around.